CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+# Contributors
```

Comment:
Do we even need a contributors file when this data is available in GitHub?

Reply:
Makes sense to me to remove all of this.

Reply:
Changed